src/main/cli/settings.c

```diff
@@ -1406,10 +1414,11 @@ const clivalue_t valueTable[] = {
     { "max7456_preinit_opu",        VAR_UINT8   | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_MAX7456_CONFIG, offsetof(max7456Config_t, preInitOPU) },
 #endif
 
-// PG_DISPLAY_PORT_MSP_CONFIG
+// PG_DISPLAY_PORT_MSP_CONFIG and PG_MSP_CONFIG
 #ifdef USE_MSP_DISPLAYPORT
     { "displayport_msp_col_adjust", VAR_INT8    | MASTER_VALUE, .config.minmax = { -6, 0 }, PG_DISPLAY_PORT_MSP_CONFIG, offsetof(displayPortProfile_t, colAdjust) },
     { "displayport_msp_row_adjust", VAR_INT8    | MASTER_VALUE, .config.minmax = { -3, 0 }, PG_DISPLAY_PORT_MSP_CONFIG, offsetof(displayPortProfile_t, rowAdjust) },
+    { "displayport_msp_serial",     VAR_INT8    | MASTER_VALUE, .config.minmax = { SERIAL_PORT_NONE, SERIAL_PORT_IDENTIFIER_MAX }, PG_MSP_CONFIG, offsetof(mspConfig_t, displayPortSerial) },
```

Comment:
Looking at this, would it make sense to add `displayPortSerial` to `PG_DISPLAY_PORT_MSP_CONFIG`? It is only usable at the moment if `USE_MSP_DISPLAYPORT` is defined anyway.

Reply:
Yeah, looks like you are right.

I'm expecting some more things to come in when we do advanced external OSDs, but we'll think about this again when they do ;)